lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -507,13 +507,12 @@ defmodule Kernel.SpecialForms do
   defmacro alias(module, opts), do: error!([module, opts])
 
   @doc """
-  Requires a given module to be compiled and loaded.
+  Tells Elixir you want to use the macros defined in a module.
```

Comment:
The structure looks good to me, but I would mention "requires" just because the term we refer to this process with is "requiring", so mentioning it is not a bad thing IMO.